kuwala/core/canvas/src/components/Modals/ExportBlockConfig/ExportBlockConfigModal.js

```diff
@@ -0,0 +1,205 @@
+        const mapParameter = (param) => {
+            let value = param.value;
+
+            if (param.type === 'date') {
+                value = moment(param.value).format('YYYY-MM-DD')
+            } else if (param.id === 'aggregated_columns') {
+                value = param.value.map(({ column, aggregation }) => `${column}KUWALA_AGG${aggregation}`);
+            }
+
+            return { ...param, value };
+        }
+
+        if (!selectedElement.data.exportBlock.exportBlockEntityId) {
+            const parsedValues = {
+                ...values,
+                parameters: values.parameters.map(mapParameter)
+            }
+            const data = {
+                export_catalog_item_id: selectedElement.data.exportCatalogItem.id,
+                input_block_ids: connectedBlocks,
+                macro_parameters: parsedValues.parameters.map((el) => {
+                    return {
+                        id: el.id,
+                        value: el.value
+                    }
+                }),
+                name: values.exportBlockName,
+                position_x: selectedElement.position.x,
+                position_y: selectedElement.position.y,
+            }
+
+            try {
+                const res = await createExportBlock(data);
+                if(res.status === 200) {
+                    const dto = new ExportBlockDTO({
+                        exportCatalogItem: selectedElement.data.exportCatalogItem,
+                        exportCatalogItemId: selectedElement.data.exportCatalogItemId,
+                        exportBlockEntityId: res.data.id,
+                        exportBlockId: selectedElement.data.exportBlock.exportBlockId,
+                        name: values.exportBlockName,
+                        connectedSourceNodeIds: selectedElement.data.exportBlock.connectedSourceNodeIds,
+                        connectedTargetNodeIds: selectedElement.data.exportBlock.connectedTargetNodeIds,
+                        isConfigured: true,
+                        macroParameters: values.parameters,
+                        columns: res.data.columns,
+                        positionX: res.data.position_x,
+                        positionY: res.data.position_y,
+                        inputBlockIds: connectedBlocks,
+                    })
+                    updateExportBlock({ addNode, elements, setElements, updatedBlock: dto });
+                    toggleExportConfigModal();
+                    setExportBlockName(values.exportBlockName);
+                }
+            } catch(e){
+                alert('Failed to create export block')
+            } finally {
+                setIsExportBlockSaveLoading(false);
+            }
+        } else {
+            // TODO: Update data block (not implemented completely in backend)
+            alert('Updating transformation blocks is currently not supported');
```

Comment:
Why is updating the export block not implemented? Seems to be fairly straightforward without a lot of side effects since it is the end of the pipeline. Please include the update functionality. Or does something speak against it at this point?